### PR TITLE
Generated Password Length reading from config.yaml

### PR DESCRIPTION
**Pull request by ramazan**

### CODE REVIEW

**Comment by yakuter** [comment]
Hello Ramazan, thank you for PR. It is a good idea to have an option about password length. However theres 2 small problem about your pull request.
1. You added .idea folder to to gitignore but you must clear your cache to make it active. There is still .idea folder and files in your pull request. Can you execute this command on your local folder:

`git rm -r --cached .`

Then you can send your request again with `git add .` etc. This will clear your repo from .idea

2. Pleas do not change config file name to config-sample in pkg/config/configuration.go file. It must be `viper.SetConfigName("config")`

Basicaly only these files should have changed about your pull request issue:
- .gitignore 
- pkg/config/configuration.go 
- store/config-sample.yml
- README.md
- login/login_helper.go

File: pkg/config/configuration.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ func Setup() {
 
 	var configuration *Configuration
 
-	viper.SetConfigName("config")
+	viper.SetConfigName("config-sample")
```
Comment (yakuter) [inline pkg/config/configuration.go:24]: Please do not change this line.

Comment (ramazan) [reply pkg/config/configuration.go]: I reverted back this change.

**Comment by ramazan** [comment]
Hi @yakuter,

thanks for comments.

i reverted back this. `viper.SetConfigName("config-sample")` 
i think its done.

**Comment by yakuter** [comment]
Great job! Thank you for your contribution :)

**Pull request merged**